src/config/quicksand.php

```diff
@@ -7,6 +7,10 @@
     // Whether to log the number of soft deleted records per model
     'log' => false,
 
+    // If you log the soft deleted records per model, this is the path where it will be stored
+    // false if you want the default laravel log file
```

Comment:
Maybe add a third line of comment showing an example path

Reply:
Sure